stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/view/internal/MessageReplyView.kt

```diff
@@ -144,10 +147,13 @@ internal class MessageReplyView : FrameLayout {
                     }
                     setTint(color)
                 }
-                isMine -> {
+                message.isMine() -> {
```

Comment:
and here

Reply:
and here as well, to show the appropriate background colour for the quoted message.